README.md

```diff
@@ -32,3 +32,328 @@ Setup and usage is demonstrated in the
 
 For developers, refer to the
 [Developers setup guide](https://open-aims.github.io/ADRIA.jl/stable/development/development_setup/).
+
+# ADRIA Docker Setup
```

Comment:
This is a lot of documentation for the main readme. I think put it in its own file, maybe under `docs/`

Reply:
Okay - moving to docker subsection of development docs